### PR TITLE
Tag non-shipping nupkg as 'NonShipping=true' in the manifest.

### PR DESCRIPTION
**Pull request by ghost**
microsoft.dotnet.msbuildsdkresolver.2.1.300-preview2-008217.nupkg should be tagged as "nonshipping" to prevent the final publishing on nuget.org

### CODE REVIEW

File: build/publish/PublishNupkgToBlobFeed.targets [hunk 1]
```diff
@@ -22,4 +22,28 @@
                     PublishFlatContainer="false"
                     SkipCreateManifest="$(IsNotOrchestratedPublish)" />
   </Target>
+
+  <Target Name="PublishNonShipNupkgToBlobFeed" Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
```
Comment (nguerrera) [inline build/publish/PublishNupkgToBlobFeed.targets]: @mmitche @dagood  Do we need to have a separate invocation of PushToBlobFeed or will it read the metadata off of ItemsToPush?

Comment (johnbeisner) [reply build/publish/PublishNupkgToBlobFeed.targets]: We do not want the metadata to apply to both nupkgs, but just the one.

Comment (mmitche) [reply build/publish/PublishNupkgToBlobFeed.targets]: I don't think so.  I think you can just add the metadata to the msbuildsdkresolver item and drastically simplify this PR.

Comment (mmitche) [reply build/publish/PublishNupkgToBlobFeed.targets]: @johnbeisner Can you just do this?

```
<ItemGroup>
  <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
</ItemGroup

<ItemGroup>
  <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg">
    <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
  </NupkgsForPublishing>
</ItemGroup
```

Comment (dagood) [reply build/publish/PublishNupkgToBlobFeed.targets]: That's correct, it uses the `ManifestArtifactData` from each item individually.

Comment (johnbeisner) [reply build/publish/PublishNupkgToBlobFeed.targets]: @mmitche
Does this work like we intend?

Comment (mmitche) [reply build/publish/PublishNupkgToBlobFeed.targets]: @johnbeisner It should, though we'll verify once this goes in.

Comment (johnbeisner) [reply build/publish/PublishNupkgToBlobFeed.targets]: Done...

File: build/Publish.targets [hunk 2]
```diff
@@ -34,7 +34,7 @@
                              GenerateChecksums;
                              UploadArtifactsToAzure;
                              UploadChecksumsToAzure;
-                             PublishNupkgToBlobFeed;
+                             PublishNonShipNupkgToBlobFeed;
```
Comment (nguerrera) [inline build/Publish.targets]: This looks like the wrong target name was kept.

File: build/publish/PublishNupkgToBlobFeed.targets [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
     <Error Condition="'$(BlobFeedCloudDropAccessToken)' == ''" Text="Missing property BlobFeedCloudDropAccessToken." />
     <Error Condition="'$(BlobFeedExpectedURL)' == ''" Text="Missing property BlobFeedExpectedURL." />
-    
+
+    <ItemGroup>
+      <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
+      <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg">
+        <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
+      </NupkgsForPublishing>
+    </ItemGroup>
```
Comment (nguerrera) [inline build/publish/PublishNupkgToBlobFeed.targets]: Why move this here? Can;t we just add the metadata to the line where this was before?

Comment (johnbeisner) [reply build/publish/PublishNupkgToBlobFeed.targets]: It should have been here all along because the itemgroup only has meaning associated with this target. That being said, I do not have a strong opinion in this matter.

Comment (nguerrera) [reply build/publish/PublishNupkgToBlobFeed.targets]: I would strongly prefer that the diff does not have unrelated changes to the task at hand. Also, it is pretty common to set items that are static outside of a target. 

On that note, please squash once this gets down to one line.

**Review by nguerrera — COMMENTED** [review]
Please make the minimal change and squash.

File: build/publish/PublishNupkgToBlobFeed.targets [hunk 4]
```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
```
Comment (nguerrera) [inline build/publish/PublishNupkgToBlobFeed.targets]: Revert the unnecessary whitespace too. The diff should show only the relevant change.

File: build/publish/PublishNupkgToBlobFeed.targets [hunk 5]
```diff
@@ -1,14 +1,17 @@
...
+    <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg">
+      <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
+    </NupkgsForPublishing>
   </ItemGroup>
 
   <Target Name="PublishNupkgToBlobFeed" Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
     <Error Condition="'$(BlobFeedCloudDropAccessToken)' == ''" Text="Missing property BlobFeedCloudDropAccessToken." />
     <Error Condition="'$(BlobFeedExpectedURL)' == ''" Text="Missing property BlobFeedExpectedURL." />
-    
+
```
Comment (nguerrera) [inline build/publish/PublishNupkgToBlobFeed.targets]: Here too.

**Review by nguerrera — APPROVED** [review]
I'll let the whitespace go. Please do squash.

**Review by mmitche — APPROVED** [review]

**Comment by nguerrera** [comment]
@dotnet-bot Ubuntu x64 Release Build

**Comment by nguerrera** [comment]
@dotnet-bot Test Ubuntu x64 Release Build
